sda-db/templates/podsecuritypolicy.yaml

```diff
@@ -30,14 +30,39 @@ spec:
     ranges:
     - min: 1
       max: 65535
-  hostNetwork: false
-  hostIPC: false
-  hostPID: false
   volumes:
   - secret
   - persistentVolumeClaim
   - configMap
   - emptyDir
   - projected
   - nfs
```

Comment:
I think only `projected`, `persistentVolumeClaim` and `emptyDir` are used by db statefulset